cmd/juju/application/deployer/series_selector_test.go

```diff
@@ -137,7 +137,7 @@ func (s *SeriesSelectorSuite) TestCharmSeries(c *gc.C) {
 				conf:                defaultSeries{},
 				supportedJujuSeries: set.NewStrings("bionic", "cosmic"),
 			},
-			err: "series not specified and charm does not define any",
+			err: `the charm defined series "precise" not supported`,
```

Comment:
why this change? if the rest of the test is unchanged, shouldn't we expect the same outcome here? why does the charm define "precise" now and it didn't before?

Reply:
The old error message was not really helpful. This PR fixes it and the new error message is more helpful and also makes more sense, isn't it?